x/programs/rust/examples/token/scripts/build_and_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+simulator_path="../../../cmd/simulator"
+simulator_bin="${simulator_path}"/bin/simulator
+
+echo "Building Simulator..."
+go build -o "${simulator_bin}" "${simulator_path}"/simulator.go
+
+export SIMULATOR_PATH="${simulator_bin}"
```

Comment:
I'm assuming these are special ENVs, can you comment that here (assuming most people will look here for a HOW-TO)